geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/utils/ThreadManagement.scala

```diff
@@ -53,9 +54,9 @@ object ThreadManagement extends LazyLogging {
     }
   }
 
-  private val executor = Executors.newSingleThreadScheduledExecutor()
+  private val executor = MoreExecutors.getExitingScheduledExecutorService(new ScheduledThreadPoolExecutor(1))
   executor.scheduleWithFixedDelay(reaper, interval, interval, TimeUnit.MILLISECONDS)
-  sys.addShutdownHook(executor.shutdown())
+  sys.addShutdownHook(executor.shutdownNow())
```

Comment:
This resolves the hanging problem for the Accumulo QuickStart, both with and without a ``-Dgeomesa.query.timeout.millis=<value>`` argument.